inc/zoo/swar/SWAR.h

```diff
@@ -250,14 +270,25 @@ struct BooleanSWAR: SWAR<NBits, T> {
         return BooleanSWAR(swar::clearLSB(this->value()));
     }
 
+    template<std::size_t N, typename = std::enable_if_t<N == SWAR<NBits, T>::Lanes, T>>
+    constexpr static BooleanSWAR fromBooleanLiterals(const bool (&args)[N]) {
```

Comment:
Come on, don't do this, I find it very ugly to change the arity of a template to apply SFINAE

Reply:
I see, check that out.